### ISSUE TITLE
How do I use a 3rd party lib required by a script tag in the html file?

### ISSUE THREAD

**Issue by NullVoxPopuli** [angular/angular-cli#1171]
I've given up with imports/requires, so I am trying the script-tag way.

in src/index.html:
```html
  {{#unless environment.production}}
  <!-- These two files (used for testing) cannot be included the 'normal' way,
  because angular-cli's build tools don't resolve dependencies of 3rd party libs. -->
  <script src="/dist/vendor/sinon/pkg/sinon.js" type="text/javascript"></script>
  <script src="/dist/vendor/axios/dist/axios.js" type="text/javascript"></script>
  {{/unless}}
</head>
```

and in angular-cli-build.js:
```js
    vendorNpmFiles: [
     ...
      'sinon/**/*.js',
    ],
```

and then in my test:
```ts
declare var sinon:any;
...
  beforeEach(() => sandbox = sinon.sandbox.create());

```

I get the error:
```
     ReferenceError: sinon is not defined
```

------------------------------

```
$ ng --version
angular-cli: 1.0.0-beta.6
node: 5.10.1
os: linux x64
```

how do I use the library?

after running `ng test`, there are 0 console errors in the opened chrome window. The errors are only in the terminal.

**Comment by RicardoVaranda** [comment]
You are calling the scripts wrong:
`<script src="/dist/vendor/axios/dist/axios.js" type="text/javascript"></script>`

Should be:
` <script src="vendor/axios/dist/axios.js" type="text/javascript"></script>`

Hope you see the difference :)

**Comment by NullVoxPopuli** [comment]
I changed the script tags in my index.html file to:

```
  <script src="vendor/sinon/pkg/sinon.js" type="text/javascript"></script>
  <script src="vendor/axios/dist/axios.js" type="text/javascript"></script>
```

my chrome has never given my 404s with this issue:
![image](https://cloud.githubusercontent.com/assets/199018/16308876/a887b8bc-3934-11e6-9b96-e20069cdfc60.png)


![image](https://cloud.githubusercontent.com/assets/199018/16308890/b5977e8e-3934-11e6-8cda-34441ac3be22.png)


In the network pane, I don't even see a call to sinon.js

![image](https://cloud.githubusercontent.com/assets/199018/16308942/f247219a-3934-11e6-8627-f8ca4f7792e1.png)

**Comment by NullVoxPopuli** [comment]
the same thing happens with jQuery, so it's non sinon specific

**Comment by filipesilva** [comment]
Closed as issue was made obsolete by #1455. You shouldn't have the issue that originally led to this approach, as webpack handles the dependencies of dependencies as you would expect it to.
